LiskHQ/lisk-desktop
# Lisk desktop tests fail when lisk-client is update beyond 5.0.1 - Closes #3804

### What was the problem?

This PR resolves #3804

### How was it solved?

- This issue is resolved with lisk-client 5.0.2
- Updated webpack to fallback os

### How was it tested?
- Run `npm run test`
- Run `npm run dev`
- Run `npm run pack`
- All of the above should work
